src/current/molt/molt-overview.md

```diff
@@ -0,0 +1,58 @@
+---
+title: MOLT Overview
+summary: Learn how to use the MOLT tools to migrate to CockroachDB.
+toc: true
+docs_area: migrate
+---
+
+This page describes the MOLT (Migrate Off Legacy Technology) tools. For more information about migrating to CockroachDB, see [Migration Overview]({% link {{site.current_cloud_version}}/migration-overview.md %}).
+
+The MOLT tools can be used when:
+
+- Converting a schema for compatibility with CockroachDB.
+- Loading test and production data into CockroachDB.
+- Validating queries on CockroachDB.
+- Performing a cutover to finalize the migration to CockroachDB.
+
+## MOLT tools
+
+<img src="{{ 'images/molt/molt_tools.svg' | relative_url }}" alt="MOLT tooling overview" style="max-width:100%" />
+
+|                        Tool                       |            Source databases           |                   Usage                   |                                      Release status                                      |
```

Comment:
Is the right term here `source databases` or `supported dialects`. Since we use dialect in a lot of other places so just wondering about consistency.

Reply:
I think `Supported databases` is good. I think we use "dialect" in body text, outside of headings.